packages/nimble-components/src/patterns/error/types.ts

```diff
@@ -1,9 +1,14 @@
 /**
- * This interface should be used by components that want to leverage the errorTextTemplate defined in templates.ts
+ * This interface should be used by components that want to leverage the error pattern
```

Comment:
nit: could we add a note to the docs explaining what the error pattern does? e.g.
```suggestion
 * This interface should be used by components that want to leverage the error pattern. The error pattern conditionally adds error text to the template when an error is configured to be visible.
```